config/auth0.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Please review available configuration options here:
+ * https://github.com/auth0/auth0-PHP#configuration-options.
+ */
+return [
+    // Should be assigned either 'api', 'management', or 'webapp' to indicate your application's use case for the SDK.
+    // Determines what configuration options will be required.
+    'strategy' => env('AUTH0_STRATEGY', \Auth0\SDK\Configuration\SdkConfiguration::STRATEGY_REGULAR),
+
+    // Auth0 domain for your tenant, found in your Auth0 Application settings.
+    'domain' => env('AUTH0_DOMAIN'),
+
+    // If you have configured Auth0 to use a custom domain, configure it here.
+    'customDomain' => env('AUTH0_CUSTOM_DOMAIN'),
+
+    // Client ID, found in the Auth0 Application settings.
+    'clientId' => env('AUTH0_CLIENT_ID'),
+
+    // Authentication callback URI, as defined in your Auth0 Application settings.
+    'redirectUri' => env('AUTH0_REDIRECT_URI', env('APP_URL') . '/callback'),
+
+    // Client Secret, found in the Auth0 Application settings.
+    'clientSecret' => env('AUTH0_CLIENT_SECRET'),
+
+    // One or more API identifiers, found in your Auth0 API settings. The SDK uses the first value for building links. If provided, at least one of these values must match the 'aud' claim to validate an ID Token successfully.
+    'audience' => \Auth0\Laravel\Configuration::stringToArrayOrNull(env('AUTH0_AUDIENCE')),
+
+    // One or more scopes to request for Tokens. See https://auth0.com/docs/scopes
+    'scope' => \Auth0\Laravel\Configuration::stringToArray(env('AUTH0_SCOPE')),
```

Comment:
We'll need to request specific scopes when fetching client access tokens to integrate with the management API (`read:clients`, `create:clients`, etc), and different ones when letting a user log in (`profile` and/or `refresh_token` and/or `email`). Can we make that distinction with this SDK?

Reply:
We will need to instantiate a separate instance of the Auth0 SDK to execute the management API calls.